src/drivers/uwb/uwb_sr150/module.yaml

```diff
@@ -13,7 +13,7 @@ parameters:
         UWB_INIT_OFF_X:
           description:
             short: UWB sensor X offset in body frame
-            long: UWB sensor positioning in relation to Drone in NED. X offset. A Positiv offset results in a Position o
+            long: UWB sensor positioning in relation to Drone in NED. X offset. A Positive offset results in a Position o
```

Comment:
Note, don't know how this sentence was supposed to end.

Reply:
Hmmm :thinking:

Reply:
Well, we're no worse off for this going in as is. Who might know the intent?

Reply:
Closing. No worse than before.